docs/items/index.md

```diff
@@ -187,6 +204,12 @@ public static final Supplier<CreativeModeTab> EXAMPLE_TAB = CREATIVE_MODE_TABS.r
 );
 ```
 
+## `ItemLike`
+
+`ItemLike` is an interface, implemented by `Item`s and [`Block`s][block] in vanilla. It defines the method `#asItem`, which returns an item representation of whatever the object actually is: `Item`s just return themselves, while `Block`s return their associated `BlockItem`.
```

Comment:
This should also mention what happens when trying to get a block with no item (I believe it returns air).